Modules/HTTPServer/Sources/HTTPServer/Server/Server.swift

```diff
@@ -157,36 +157,46 @@ extension Server {
     }
 
     public func process(stream: Stream) throws {
-        let parser = RequestParser(stream: stream, bufferSize: bufferSize)
+        let chunkBytes = UnsafeMutablePointer<UInt8>.allocate(capacity: 4096)
```

Comment:
maybe we should expose this as a configuration? could be done later. maybe add a `// Todo:`?